docs/product-spec.md

```diff
@@ -21,52 +21,52 @@ By offering simple, readily available, off-the-shelf hardware and
 robust software, we can enable 3 election staffers to manage mailing
 and receiving 50,000 ballots.
 
-
 ## VxMail
 
 We propose VxMail, a set of tools to help small jurisdictions quickly scale
 up their vote-by-mail capacity.
 
-### Components
+VxMail includes the following components:
 
-VxMail includes:
-* a Ballot Manager to create and proof mail ballots, schedule printing
-  and mailing, log receipt of returned ballots, and verify voter
+- **VxMail Ballot Manager** is used to create and proof ballots, schedule
```

Comment:
I'd like to not start giving them official names, as we'd discussed. Let's stick to just `Ballot Manager`.
```suggestion
- a **Ballot Manager** is used to create and proof ballots, schedule
```

Reply:
Cool. I just want the doc to be consistent.